modules/core/src/test/java/org/apache/ignite/internal/managers/discovery/IgniteTopologyPrintFormatSelfTest.java

```diff
@@ -62,237 +64,209 @@
 
     /** {@inheritDoc} */
     @Override protected void beforeTest() throws Exception {
+        super.beforeTest();
+
         stopAllGrids();
     }
 
     /** {@inheritDoc} */
     @Override protected void afterTest() throws Exception {
         stopAllGrids();
 
-        if (log instanceof MockLogger)
-            ((MockLogger)log).clear();
+        super.afterTest();
     }
 
     /**
      * @throws Exception If failed.
      */
     @Test
     public void testServerLogs() throws Exception {
```

Comment:
Here and below. What about adding few words within test comment about what test is supposed to do?